src/lib.rs

```diff
@@ -171,6 +171,24 @@ pub fn levenshtein(a: &str, b: &str) -> usize {
     result
 }
 
+/// Calculates normalized score of the Levenshtein algorithm
+///
+/// ```
+/// use strsim::normalized_levenshtein;
+///
+/// assert!((normalized_levenshtein("kitten", "sitting") - 0.57142).abs() < 0.00001);
+/// assert!((normalized_levenshtein("", "") - 1.0).abs() < 0.00001);
+/// assert!(normalized_levenshtein("", "second").abs() < 0.00001);
+/// assert!(normalized_levenshtein("first", "").abs() < 0.00001);
+/// assert!((normalized_levenshtein("string", "string") - 1.0).abs() < 0.00001);
+/// ```
+pub fn normalized_levenshtein(a: &str, b: &str) -> f64 {
+    if a.len() == 0 && b.len() == 0 {
```

Comment:
yeah, you're right. `bytes` != `chars` in general case

btw, I just found this method [`&str::is_empty`](https://doc.rust-lang.org/stable/std/primitive.str.html#method.is_empty). For me it looks like even better way here. Yes, it returns `true` for zero **bytes**, but can any string containing multi-byte chars be an empty string at same time?

One benefit I can see here, we don't measure `length` of string, but ask "is this string empty?" Which has more clear semantics

What do you think?

Reply:
1. Good idea. I agree that using `is_empty` would have better semantics for this line. Go for it. We'll still have to count the chars for the return expression though.

2. Yeah, I've seen that bit about graphemes before, but I'd rather keep it simple for now. We'd have to use an external crate for it, and I wouldn't be surprised if it's slower than character iteration. Maybe in the future, we can add a way for users to specify the iteration method.

Reply:
Fixes:
- check empty strings by` is_empty` in 13b89d3ecd931b9e131d3a28c7727f0fd19ef2f8
- count chars instead of bytes in 601d4cc86517da781d629fa74a0a63407af19015